lib/src/gotrue_client.dart

```diff
@@ -299,4 +335,12 @@ class GoTrueClient {
   void _notifyAllSubscribers(AuthChangeEvent event) {
     stateChangeEmitters.forEach((k, v) => v.callback(event, currentSession!));
   }
+
+  Future openPersistSessionDb() async {
+    if (!_isPersistSessionDbOpen) {
+      _persistSessionDb = await getDatabaseFactory().openDatabase(
+          '${path.getBasePath()}/${Constants.persistSessionDbFileName}');
+      _isPersistSessionDbOpen = true;
+    }
+  }
```

Comment:
It will be better to have storage class with 3 public methods:
- write to db
- read from db
- delete from db